ios/RNGoogleCast/RNGoogleCast.m

```diff
@@ -25,6 +25,10 @@ - (instancetype)init {
   return self;
 }
 
++ (BOOL)requiresMainQueueSetup {
+  return YES;
+}
```

Comment:
lol, this is to remove the yellow notification! I thought it was way more complicated